API/ev3_lcd.c

```diff
@@ -1986,5 +1988,5 @@ void TermPrintln(const char *fmt, ...)
 void Ev3Clear()
 {
 	LcdClean();
-	LcdPrintf("%s", "\f");
+	LcdPrintf(1, "%s", "\f");

```

Comment:
Could you reorder the commits to squash these fixups?

Reply:
Good point, I squashed multiple commits that fixed bugs introduced by previous new commits.